setup-ovs.sh

```diff
@@ -34,23 +50,15 @@ if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
      esac
   done
   echo -e "default dev: $default_device ($profile_name)\nsecondary dev: $secondary_device ($secondary_profile_name)"
-  
+
   mac=$(sudo nmcli -g GENERAL.HWADDR dev show $default_device | sed -e 's/\\//g')
 
-  # delete old bridge if it exists
   ovs-vsctl --if-exists del-br brcnv
-  
-  # make bridge
+  # code.engineering
   nmcli conn add type ovs-bridge conn.interface brcnv
   nmcli conn add type ovs-port conn.interface brcnv-port master brcnv
-  nmcli conn add type ovs-interface \
-                 conn.id brcnv-iface \
-                 conn.interface brcnv master brcnv-port \
-                 ipv4.method auto \
-                 ipv4.dhcp-client-id "mac" \
-                 connection.autoconnect no \
-                 802-3-ethernet.cloned-mac-address $mac
-  
+  nmcli conn add type ovs-interface conn.id brcnv-iface conn.interface brcnv master brcnv-port ipv4.method auto ipv4.dhcp-client-id "mac" connection.autoconnect no 802-3-ethernet.cloned-mac-address $mac
+
   # make bond
   nmcli conn add type ovs-port conn.interface bond0 master brcnv ovs-port.bond-mode balance-slb
   nmcli conn add type ethernet conn.interface $default_device master bond0
```

Comment:
can we set the slave names to bet consistent? this way we can just iterate them when setting them to up in the end
```
nmcli conn add type ethernet conn.interface $default_device master bond0 con-name ovs-slave-1
nmcli conn add type ethernet conn.interface $secondary_device master bond0 con-name ovs-slave-2
```

Reply:
ditto? 
If we decide to change the names then i will be for all interfaces related to OVS right?

Reply:
I'm good with changing just the ones with dynamic names (like ovs-slave-<original-interface-name>)
but if you think we should change all of them I'm good with it as well.

Reply:
Changed all names to static ones, all names start with OVS-...